openmetadata-ui/src/main/resources/ui/src/utils/IngestionWorkflowUtils.ts

```diff
@@ -126,9 +130,7 @@ export const getSchemaByWorkflowType = (
       break;
 
     default:
-      schema = {};
-
-      break;
+      return schema;
```

Comment:
Why we removed empty assignment for schema here?

Reply:
The change was made to return an empty object in case no match is found.Earlier it wasn't returning an empty object causing a custom jest case to fail but with this change it returns an empty object making the test case pass

Reply:
Reverted back the changes